src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum LottoRank {
+
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 1_500_000),
+    THIRD(4, 50_000),
+    FOURTH(3, 5_000);
+
+    private int countOfMatch;
+    private int winningMoney;
```

Comment:
재할당이 불가능 하도록 final 을 사용해 보면 어떨까요? 😄 
```suggestion
    private final int countOfMatch;
    private final int winningMoney;
```